Patch

RickBakkr opened the pull request:
This patch includes:
- removal of HTML tags from JS variables (used in Typed.js)
- a minor UI improvement: use of a styled "p" element rather than a header
- Improved consistency usage of ' and ". Double-quotes should be used 

Alex, feel free to merge or close the PR.

RickBakkr committed:
Typed without HTML tags in the Javascript. Minor UI improvement

RickBakkr committed:
Consistent use of ' and " in the HTML

unixfy commented:
- The `<b>` tags in Typed.js are intended, we use them to make every statement bold, *excluding the periods*.

- Please use individual classes to modify font-size of text

DON'T:
```
p {
font-size: 0.8em;
}
```

DO:
```
.text-class {
font-size: 0.8em;
}
```

We often have several elements with the same type (e.g. `p`) so using CSS to modify them all often causes unintended changes.

Apart from that, I will be merging the changes in a moment.

Thanks :+1:

unixfy committed:
Fixed TypedJS and added class to <p>

Added back the bold in the TypedJS strings and added an `about-text` class to the About element so we can change its font size (instead of  using h5)

unixfy committed:
Made font-size affect class instead of p element

unixfy reviewed (approved):
I've edited your changes to make them compatible with the points I specified in my comment earlier.
